docs/Hosts/Spawn-Hosts.md

```diff
@@ -50,14 +50,34 @@ Clicking it will pre-populate the spawn host page with a request to spawn a host
 
 ![spawn_host_modal.png](../images/spawn_host_modal.png)
 
-Fetching artifacts can also be performed manually; see [fetch](../CLI#fetch) in the Evergreen command line tool documentation.
+The spawn host will not fetch the artifacts completely automatically; you will need to run the following command (which will prompt you to authenticate):
```

Comment:
can we add a brief explanation for the motivation behind this behavior?

Reply:
I added a motivation/reasoning, please lmk if it's clear! Docs are hard